realm/realm-library/src/main/java/io/realm/MixedOperator.java

```diff
@@ -99,17 +98,19 @@ protected MixedOperator(MixedType type, NativeMixed nativeMixed) {
 
     abstract <T> T getValue(Class<T> clazz);
 
-    MixedType getType(){
+    MixedType getType() {
         return this.type;
     }
 
     Class<?> getTypedClass() {
         return type.getTypedClass();
     }
 
-    boolean coercedEquals(MixedOperator mixedOperator){
+    boolean coercedEquals(MixedOperator mixedOperator) {
         return getNativeMixed().coercedEquals(mixedOperator.getNativeMixed());
     }
+
+    public void checkValidObject(BaseRealm realm) { }
```

Comment:
Is it right that this should be a no-op?

Reply:
It is for primitive types, on RealmModelMixedOperator it throws if the object is not valid.